File: src/main/java/domain/model/airflow/AirflowEstimation.java
package domain.model.airflow;

import java.util.List;

public class AirflowEstimation {
    public final List<List<Double>> estimatedAirflowGramsPerSecondLogs;
    public final List<List<Double>> measuredAirflowGramsPerSecondLogs;
    public final List<List<Double>> measuredRpmLogs;

    public AirflowEstimation(List<List<Double>> estimatedAirflowGramsPerSecondLogs, List<List<Double>> measuredAirflowGramsPerSecondLogs, List<List<Double>> measuredRpmLogs) {
        this.estimatedAirflowGramsPerSecondLogs = estimatedAirflowGramsPerSecondLogs;
        this.measuredAirflowGramsPerSecondLogs = measuredAirflowGramsPerSecondLogs;
        this.measuredRpmLogs = measuredRpmLogs;
    }
}

File: src/main/kotlin/domain/model/airflow/AirflowEstimation.kt
package domain.model.airflow

data class AirflowEstimation(
    val estimatedAirflowGramsPerSecondLogs: List<List<Double>>,
    val measuredAirflowGramsPerSecondLogs: List<List<Double>>,
    val measuredRpmLogs: List<List<Double>>
)
